pom.xml

```diff
@@ -260,12 +230,5 @@
     </pluginRepository>
   </pluginRepositories>
 
-  <distributionManagement>
```

Comment:
It is actually required. Otherwise you won't be able to build the module if there is no parent POM in the local cache

Reply:
Are you maybe thinking of `<repositories>`?

Just checked with Jesse as well, since he added a [`TODO`](https://github.com/jenkinsci/analysis-pom-plugin/pull/5/files#diff-600376dffeb79835ede4a0b285078036R264) to his PR to remove the same block.

Reply:
It should